src/coreclr/pal/src/include/pal/palinternal.h

```diff
@@ -229,6 +229,24 @@ function_name() to call the system's implementation
 #define tanf DUMMY_tanf
 #define tanhf DUMMY_tanhf
 #define truncf DUMMY_truncf
+#define acos DUMMMY_acos
```

Comment:
Why do we still need these #defines ?

Reply:
We need these defines as we now expose the CRT functions (with their CRT implementations) through the PAL headers (as we can't use the CRT headers outside of the PAL). But since we use the CRT headers in the PAL, we need to use this dummy define for one of the signatures to avoid conflicts.

Once we can include the CRT headers, we can remove all of this.